servicex/models.py

```diff
@@ -40,13 +40,14 @@ class UserModel(db.Model):
     __tablename__ = 'users'
     admin = db.Column(db.Boolean, default=False)
     created_at = db.Column(DateTime, default=datetime.utcnow)
-    id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(320), nullable=False, unique=True)
-    full_name = db.Column(db.String(120), nullable=False)
+    experiment = db.Column(db.String(120))
+    id = db.Column(db.Integer, primary_key=True)
     institution = db.Column(db.String(120))
-    key = db.Column(db.String(120), nullable=False)
+    name = db.Column(db.String(120), nullable=False)
     pending = db.Column(db.Boolean, default=True)
-    experiment = db.Column(db.String(120))
+    refresh_token = db.Column(db.Text, nullable=False, unique=True)
+    sub = db.Column(db.String(120), nullable=False, unique=True, index=True)
```

Comment:
I know this name makes sense in the JWT world, but maybe it should be renamed to `subject` or `jwt_subject`, or just plain `identity`

Reply:
Seems like it's really an OAuth term.. apparently used by OAuth providers who adhere to the OpenID Connect standard:

>The OpenID Connect ID Token is a signed JSON Web Token (JWT) that is given to the client application along side the regular OAuth access token. The ID Token contains a set of claims about the authentication session, including an identifier for the user (sub), the identifier for the identity provider who issued the token (iss), and the identifier of the client for which this token was created (aud).

I'll rename it to `oauth_subject` (I don't want to confuse it with `id` which is our ServiceX user ID, an integer). If we wanted to support providers other than Globus down the road and those providers also complied with OIDC (as CILogon does) it probably wouldn't be too bad.